docker_build/opt/cosbench/conf/manta-config.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <workload name="manta-sample" description="sample benchmark for manta">
 
-  <storage type="manta" config="chunked=false;manta.retries=0;manta.verify_uploads=false;" />
+  <storage type="manta" config="chunked=false;manta.retries=0;manta.verify_uploads=false;test_type=buckets" />
```

Comment:
I suggest adding a comment above this line to give a hint about tweaking the `test_type` configuration value or really all of the values. Perhaps give a link to the documentation?

Reply:
Noted. Added a comment explaining the use of the new parameter.